src/gbd_mapping_generator/data.py

```diff
@@ -48,11 +48,12 @@ def get_risks():
 
 
 def get_covariates():
-    covariates = gbd.get_covariate_metadata().reset_index(drop=True)
-    covariates = pd.DataFrame({'covariate_name': clean_entity_list(covariates.covariate_name),
-                               'covariate_id': covariates.covariate_id})
-    return covariates.sort_values('covariate_id')
+    data = get_covariate_data()
```

Comment:
what changes is this? Doesn't seem like the singular/plural naming or the scrubbing of '&'. Was that function get_covariate_data() there before but just unused ?